include/vrv/toolkit.h

```diff
@@ -195,6 +195,13 @@ class Toolkit {
      */
     int GetTimeForElement(const std::string &xmlId);
 
+    /**
+     * Return the MIDI pitch of the element with the ID (xml:id).
+     * RenderToMidi() must be called prior to using this method.
+     * Returns -1 if no element is found.
+     */
+    int GetMIDIPitchForElement(const std::string &xmlId);
```

Comment:
Change it to GetMIDIValuesForElement and return a string / json object with for now only the pitch. We can then use the same method if one day we want to add other MIDI values

Reply:
Sounds good

Reply:
Done, ready to pull.